### PR TITLE
introduce some basic documentation as well as all the necessary files…

### PR DESCRIPTION
**Pull request by Gleethos**
I covered the basics of how to set everything up. 
I think there might be some more to add, but for now I think this convers the basics.

I would love to discuss some more details with you and maybe show you how to do it all...
Maybe in person next time we meet?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,85 @@ SpockReact is an easy way to automatically generate documentation from Spock tes
...
+whatever location you want.
+This is also the location where you will want the Spock-React website
+to look for the json files so that it can read them and generate
+the documentation!
+
+You have to specify this same location in the form 
+of a URL in the `environment.json` file.
+So if you host your documentation on GitHub, you might have a URL
+looking something like this: `https://raw.githubusercontent.com/user/proj-reo/master/docs/spock/reports`
+
```
Comment (Sonatai) [inline README.md:85]: Please add a TODO for the frontend part => also part of getting started and will be added later

Comment (Gleethos) [reply README.md:99]: I am not sure, where exactly should I add "TODO"?

Comment (Sonatai) [reply README.md:99]: add the end of the "getting started" section. You have to setup the frontend too 🤗

Comment (Gleethos) [reply README.md:99]: >  You have to setup the frontend too 
What do you mean by that?

Comment (Gleethos) [reply README.md:99]: I added frontend as title and a TODO underneath

**Pull request merged**